### ISSUE TITLE
ipdf - rst docs don't build (Trac #1297)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1291]
`i3_project()` is missing the `DOCS_DIR` argument

Migrated from https://code.icecube.wisc.edu/ticket/1297
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "`i3_project()` is missing the `DOCS_DIR` argument", 
    "reporter": "nega", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "ipdf - rst docs don't build", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-08-28T19:47:51", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150831T04:58:43: 
* olivas changed owner from "claudio.kopper" to "olivas"
* olivas changed milestone from "" to "IceRec V5"
* olivas changed status from "new" to "assigned"
* olivas changed component from "cmake" to "icerec"
* olivas changed priority from "normal" to "blocker"

**Comment by icecube-trac** [comment]
Trac update at 20150908T19:48:18: olivas changed status from "assigned" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20150908T20:18:02: 
* olivas commented:

> In changeset 137358:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="137358"
> docs build with meta-project docs.  closes #1291.
> ```


* olivas changed status from "accepted" to "closed"
* olivas changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas changed milestone from "IceRec V5" to ""
* olivas commented:

> Milestone IceRec V5 deleted
